src/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  width: 200px;
+  display: flex;
+  flex-wrap: wrap;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.imac {
+  height: 194px;
+}
+
+.imac img {
+  max-width: 160px;
+  height: 134px;
+  margin: 30px 20px;
+}
+
+.description {
+  margin: 0 15px;
+}
+
+.description p {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 15px 0;
+  margin-bottom: 0;
+}
+
+#itemCode {
```

Comment:
IDs are used for JS, try not to style the id if you can avoid it. This makes impossible to reuse the code. Stylize classes. [Style guide](https://mate-academy.github.io/style-guides/htmlcss.html#id-selectors)